exceptions/does_not_exist.py

```diff
@@ -207,26 +193,21 @@ class MemberRoleDoesNotExistError(RoleDoesNotExistError):
 
     # noinspection PyMethodParameters,PyPep8Naming
     @classproperty
-    def ERROR_CODE(cls) -> str:  # noqa: N802,N805
-        """The unique error code for users to tell admins about an error that occurred."""  # noqa: D401
+    @override
+    def ERROR_CODE(cls) -> str:  # noqa: N805
         return "E1023"
 
     # noinspection PyMethodParameters,PyPep8Naming
     @classproperty
-    def DEPENDENT_COMMANDS(cls) -> frozenset[str]:  # noqa: N802,N805
-        """
-        The set of names of bot commands that require this Discord entity.
-
-        This set being empty could mean that all bot commands require this Discord entity,
-        or no bot commands require this Discord entity.
-        """  # noqa: D401
+    @override
+    def DEPENDENT_COMMANDS(cls) -> frozenset[str]:  # noqa: N805
         # noinspection SpellCheckingInspection
```

Comment:
what's the logic behind removing all this documentation (not this one specififcally but all of it)

Reply:
Due to the use of the `@override` decorator the documentation should get inherited from the parent class. In all these cases the child class docstrings were replicas of the parent class's docstrings so just caused more places for updates to need to be made. I'm not sure about VSCode, but certainly when PyCharm identifies an `@override`n method, hovering over that method will display the docstring of the parent class.

Reply:
ah yeah vsc does the same, fairs

Reply:
Phew. That wouldve been more irritating if we had to accommodate a missing feature with worse code.